src/tokens/selectors.ts

```diff
@@ -419,12 +419,7 @@ export const cashOutTokensByNetworkIdSelector = createSelector(
       showZeroBalanceTokens,
   ],
   (tokens, showZeroBalanceTokens) =>
-    tokens.filter(
-      (tokenInfo) =>
-        ((showZeroBalanceTokens ? tokenInfo.showZeroBalance : false) ||
-          tokenInfo.balance.gt(TOKEN_MIN_AMOUNT)) &&
-        tokenInfo.isCashOutEligible
-    )
+    tokens.filter((tokenInfo) => showZeroBalanceTokens || tokenInfo.isCashOutEligible)
```

Comment:
cKES isn't marked as a showZeroBalanceToken in address-metadata but should be shown here. Just removed the balance check since both cKES and cUSD should be shown for 0 balance and these are the only tokens